docs/13_upload.md

```diff
@@ -1,48 +1,35 @@
-# Creating Dandisets and Uploading Data
+# Create Dandisets and Upload Data
 
-To create a new Dandiset and upload your data, you need to have a DANDI account.
+You can create a new Dandiset at [http://dandiarchive.org](http://dandiarchive.org). 
+This Dandiset can be fully public or embargoed according to NIH policy.
+When you create a Dandiset, a permanent Dandiset ID is automatically assigned to it.
 
-## Create an Account on DANDI
-
-To create a DANDI account:
-
-1. [Create a GitHub account](https://github.com/) if you don't have one.
-2. Using your GitHub account, [register a DANDI account](https://gui.dandiarchive.org/#/user/register).
-
-You will receive an email acknowledging activation of your account within 24
-hours, after which you can log in to DANDI using GitHub by clicking the login
-button.
-
-## Create a Dandiset and Add Data
-
-You can create a new Dandiset at https://dandiarchive.org. This Dandiset can be fully 
-public or embargoed 
-according to NIH policy.
-When you create a Dandiset, a permanent ID is automatically assigned to it.
 To prevent the production server from being inundated with test Dandisets, we encourage developers to develop 
-against the development server (https://gui-staging.dandiarchive.org/). Note 
+against the development server ([https://gui-staging.dandiarchive.org/](https://gui-staging.dandiarchive.org/)). Note 
 that the development server
```

Comment:
While at it may be just make it all one sentence per line? ;-)
```suggestion
against the development server ([https://gui-staging.dandiarchive.org/](https://gui-staging.dandiarchive.org/)). 
Note that the development server
```